src/main/java/seedu/address/logic/parser/FlashBackParser.java

```diff
@@ -6,23 +6,10 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.AddCommand;
-import seedu.address.logic.commands.ClearCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FilterCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.RedoCommand;
-import seedu.address.logic.commands.ReviewCommand;
-import seedu.address.logic.commands.SortCommand;
-import seedu.address.logic.commands.StatsCommand;
-import seedu.address.logic.commands.UndoCommand;
-import seedu.address.logic.commands.ViewCommand;
+import seedu.address.logic.commands.*;
```

Comment:
I think wildcard import is not allowed

Reply:
Thanks for the catch, will update accordingly